REQUIRE

```diff
@@ -11,3 +11,7 @@ ProgressMeter
 RecipesBase
 Reexport
 Suppressor
+Requires
```

Comment:
Let us not require `Requires` and also remove the file `require.jl` if you do not need it here.